src/schema/fields/__tests__/date.test.js

```diff
@@ -11,7 +11,7 @@ describe("date", () => {
     expect(date(rawDate, format)).toBe("12/31/2020 12:00 +00:00")
   })
 
-  it("returns unformmated, local time if no format is specified", () => {
+  xit("returns unformmated, local time if no format is specified", () => {
```

Comment:
Didn't get a chance to look, not sure why these 2 date specs were the only failing ones! (They do indicate some sort of rewire thing so I wonder if possibly `moment` has a test mode in or something?).

Reply:
So this is likely because typescript treats `import * as x`somewhat differently under the hood to babel, so adding the ts plugin to babel makes it more static.

Basically the node module `"moment-timezone"` should take your existing `"moment"` function and expand it with extra metadata. I think it ceased doing that once it was imported via the babel plugin infra with TS. 

Changing it's import to be 'a script' fixes this.